adapters/handlers/rest/configure_api.go

```diff
@@ -591,6 +593,17 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 	}
 
 	appState.SchemaManager = schemaManager
+
+	// Initialize usage service after all components are ready
+	if appState.Modules.UsageEnabled() {
```

Comment:
can this be moved to the method where we configure all modules? so that everything related to modules will be in one place?